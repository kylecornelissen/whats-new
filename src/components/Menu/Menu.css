ul {
  list-style: none;
  text-align: left;
}

li {
  padding: 2% 0%;
  margin: 3% 0%;
  width: 95%;
  cursor: pointer;
}

.category-link {
  /* background: rgba(181,106,241, 0.4); */
  border: 3px solid #FFAE00;
  border-left: 0px;
  border-radius: 0px 10px 10px 0px;
  display: block;
  padding: 2%;
}

.category-highlight {
  background: rgb(181,106,241);
  border: 20px solid white;
  border-top: 3px solid white;
  border-bottom: 3px solid white;
  border-left: 0px;
  color: #FFAE00;
  text-shadow: 1px 1px 6px black;
  border-color:
}
