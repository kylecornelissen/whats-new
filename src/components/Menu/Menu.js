import React from 'react';
import './Menu.css'

const Menu = (props) => {
  return (
    <ul>
      <li>Local News</li>
      <li>Technology</li>
      <li>Entertainment</li>
      <li>Science</li>
      <li>Health</li>
    </ul>
  )
}

export default Menu;
