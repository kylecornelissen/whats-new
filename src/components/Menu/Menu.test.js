import React from 'react';
import { shallow } from 'enzyme';
import Menu from './Menu';

describe('Menu', () => {
  it('should match a snapshot of the Menu', () => {
    const wrapper = shallow(<Menu changeCategory={jest.fn()}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
