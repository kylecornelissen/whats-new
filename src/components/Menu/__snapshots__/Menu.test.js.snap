// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should match a snapshot of the Menu 1`] = `
<aside>
  <ul>
    <li
      key="local"
    >
      <a
        className="category-link"
        onClick={[Function]}
      >
        Local News
      </a>
    </li>
    <li
      key="technology"
    >
      <a
        className="category-link"
        onClick={[Function]}
      >
        Technology
      </a>
    </li>
    <li
      key="entertainment"
    >
      <a
        className="category-link"
        onClick={[Function]}
      >
        Entertainment
      </a>
    </li>
    <li
      key="science"
    >
      <a
        className="category-link"
        onClick={[Function]}
      >
        Science
      </a>
    </li>
    <li
      key="health"
    >
      <a
        className="category-link"
        onClick={[Function]}
      >
        Health
      </a>
    </li>
  </ul>
</aside>
`;
